sycl/doc/GettingStartedXilinxFPGA.md

```diff
@@ -119,21 +118,22 @@ bitstream.
 
 For this, download from
 https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/vitis.html
-the "Xilinx Vitis 2020.2: All OS installer Single-File Download"
+the "Xilinx Unified Installer 2021.1 SFD"
 somewhere you have enough storage.
 
 Create a `/opt/xilinx` with the right access mode so you can work in
 it.
 
 Materialize the installer files with
 ```bash
-tar zxvf Xilinx_Unified_2020.2_1118_1232.tar.gz
+cd /opt/xilinx
+tar zxvf .../Xilinx_Unified_2021.1_0610_2318.tar
 ```
 
 Since the graphics Java installer does not work on modern Linux
 distributions like Ubuntu 20.10, use the batch-mode version:
 ```bash
-Xilinx_Unified_2020.2_1118_1232/xsetup --batch Install --location /opt/xilinx --agree 3rdPartyEULA,WebTalkTerms,XilinxEULA --edition "Vitis Unified Software Platform"
+/opt/xilinx/Xilinx_Unified_2021.1_0610_2318/xsetup --batch Install --location /opt/xilinx --agree 3rdPartyEULA,WebTalkTerms,XilinxEULA --edition "Vitis Unified Software Platform"
```

Comment:
That is funny because last Friday I installed Vitis 2021.1 on my laptop and I run out of space, so I had to move all the un-tarred files anyway somewhere else...